src/main/resources/io/jenkins/plugins/sprp/YAML_BranchProjectFactory/config.jelly

```diff
@@ -24,8 +24,16 @@ THE SOFTWARE.
 -->
 
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+<j:jelly xmlns:j="jelly:core" xmlns:c="/lib/credentials" xmlns:f="/lib/form">
     <f:entry title="${%Script Path}" field="scriptPath">
         <f:textbox default="Jenkins.yaml"/>
     </f:entry>
+
+    <f:entry title="${%Git Credentials}" field="credentialsId">
```

Comment:
Why does it need to be here?  IIUC it restricts the SCM support options.

Reply:
I think I should remove this thing, as this will not be used.